core/zts/src/main/rdl/RoleCert.rdli

```diff
@@ -16,3 +16,16 @@ resource RoleCertificate POST "/rolecert" (name=PostRoleCertificateRequestExt) {
         ResourceError UNAUTHORIZED;
     }
 }
+
+// Fetch all roles that are tagged as requiring role certificates for service
+resource RoleList GET "/rolecert?service={service}" (name=getRolesRequireRoleCert) {
+    ResourceName service;
```

Comment:
this should be:

EntityName principal;

Reply:
Fixed